docs/api.md

```diff
@@ -6,6 +6,206 @@ challenges presented by JavaScript and web browsers.
 Please direct clarifications or observations of inaccuracy to [our issue
 tracker](https://github.com/iodide-project/iodide/issues/new).
 
+
+## `iodide.file`
+
+The `iodide.file` API provides convenience functions around working with files
+uploaded to the Iodide server in your notebook.
+
+
+
+### `iodide.file.save(data, fileName[, saveOptions])`
+
+Returns a
+[Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises),
+which, when resolved, will signal that `data` was uploaded to the server under
+the filename `fileName`.
```

Comment:
If it errors, I assume it calls the `Promise's` `reject`?

Reply:
Good catch! It does, and I'll make sure to add this to the docs.